src/main/java/ca/bc/gov/ag/efax/mail/service/parser/EmailParser.java

```diff
@@ -49,10 +49,10 @@ public DocumentDistributionMainProcessProcessResponse parse(EmailMessage emailMe
             logger.error("Unrecognized email, \nsubject: [{}]", subject);
         }
 
-        // Default error to FAXListenFault if a jobId was found, but could not parse a status.
+        // Default error to FAXSendFault if a jobId was found, but could not parse a status.
```

Comment:
Do we need to update the error on line 49 to have more details?

Reply:
Possibly. I had more before, but removed it because I wanted to be careful about putting PI in the logs.